scripts/tests/test_suites.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+declare -a test_array="($(find src/app/tests/suites -type f -name "*.yaml" -exec basename {} .yaml \;))"
+
+echo "Found tests:"
+for i in "${test_array[@]}"; do
+    echo "   $i"
+done
+echo ""
+echo ""
+
+for i in "${test_array[@]}"; do
+    echo "  ===== Running test: $i"
+    echo "          * Starting cluster server"
+    rm -rf /tmp/chip_tool_config.ini
+    sleep 1
```

Comment:
Ideally we should avoid having "sleep" calls anywhere in the code. This seems to indicate we have some racing issues here.

Reply:
Yep.  We do; see all the TODO comments.  I have PRs up to fix the races (#7428 and #7430) and a PR up at #7433 that removes the sleeps and another at #7434 to remove the TODO workarounds.  Reviews on those would be much appreciated, @andy31415 !